README.md

```diff
@@ -34,6 +34,23 @@ The tile object will have following format:
 }
 ```
 
+## Environment variables needed to run the integration tests locally
+
+Look at the codeship settings to get the values for these variables. Either export them into your terminal or save them in an `.env` file.
+
+```
+AWS_S3_SEARCH_RESULT_BUCKET
+S3_TAGGY_BUCKET
+DYNAMO_DB_TABLE
+SEARCH_RESULT_TOPIC
+AWS_REGION
+```
```

Comment:
Consider adding a link to: https://codeship.com/projects/147193/configure_environment in the readme to help people get the environment variables. 👍